server/addVideo.js

```diff
@@ -0,0 +1,27 @@
+const db = require("./db/db-connection.js");
+
+async function addVideo(videoId) {
+  const API_KEY = process.env.YOUTUBE_API_KEY;
```

Comment:
nice! awesome job applying what we talked about. much cleaner for your frontend now to not have to think bout this part